docs/features/shared-volumes-with-NFS.md

```diff
@@ -0,0 +1,207 @@
+# Supporting shared volumes with vSphere Docker Volumes
+
+
+## Requirements
+* Developer should able to use externally managed NFS share as “shared docker volume” using “vsphere” volume driver.
+* This allows them to manage NFS directories as docker volumes using Docker APIs, Compose etc 
+* Extend the vSphere Docker Volume plugin to support shared volumes
+* Volumes are accessible on multiple docker hosts running on  multiple ESX hosts in a Docker swarm
+* Allow user to specify what remote dirs are managed by the plugin
+* Support all current docker volume plugin API for shared volumes
+
+##  Assumptions
+* Administrator sets up shares on NFS server(s) and ensures Docker Hosts has permissions to mount and modify the remote dirs.
+* Developer responsible for (one-time) plugin configuration.
+* 
+
+
+## vSphere Docker Volume Plugin Drivers – User experience
+
+  1. Setup
+      a. Set up Plugin Config
+      b. Restart plugin
+  2. Using shared volumes
+      a. docker volume create …
```

Comment:
Sure, the format isn't changed except to add network FS instances in the config and optionally in the command line. All other commands remain the same. But I'll add examples for each.